website/content/en/docs/contributing/development-guide.md

```diff
@@ -158,8 +158,8 @@ Finally, to deploy the correct IAM permissions, including the instance profile f
 make setup
 ```
 
-## Profiling memory
-Karpenter exposes a pprof endpoint on its metrics port.
+## Profiling
+Karpenter exposes a pprof endpoint on its metrics port when [profiling](https://karpenter.sh/docs/reference/settings/) is enabled.
```

Comment:
You should be able to use a relref rather than an absolute link. This will ensure the link will route to the site-under-test for preview and local development rather than the public site. 
```suggestion
Karpenter exposes a pprof endpoint on its metrics port when [profiling]({{< relref "../reference/settings" >}}) is enabled.
```